docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
I doubt anyone will build the docs with windows so we can exclude the `make.bat` file